test-network-k8s-operator/justfile

```diff
@@ -0,0 +1,191 @@
+ORDERER_IMAGE_TAG   := env_var_or_default("TEST_NETWORK_ORDERER_IMAGE_TAG", FABRIC_VERSION)
+CHANNEL_NAME        := env_var_or_default("TEST_NETWORK_CHANNEL_NAME",      "mychannel")
+CHAINCODE_NAME      := env_var_or_default("TEST_NETWORK_CHAINCODE_NAME",    "asset-transfer")
+CHAINCODE_VERSION   := env_var_or_default("TEST_NETWORK_CHAINCODE_VERSION", "v0.1.4")
+CHAINCODE_SEQUENCE  := env_var_or_default("TEST_NETWORK_CHAINCODE_SEQUENCE","1")
+CHAINCODE_PKG_NAME  := env_var_or_default("TEST_NETWORK_CHAINCODE_PKG_NAME","asset-transfer-typescript-v0.1.4.tgz")
+CHAINCODE_PKG_URL   := env_var_or_default("TEST_NETWORK_CHAINCODE_PKG_URL", "https://github.com/hyperledgendary/full-stack-asset-transfer-guide/releases/download/v0.1.4/" + CHAINCODE_PKG_NAME)
+
+
+###############################################################################
+# KIND / k8s targets
+###############################################################################
+
+# Start a local KIND cluster with nginx ingress
+kind: check unkind
+    scripts/kind_with_nginx.sh {{CLUSTER_NAME}}
+
+# Shut down the KIND cluster
+unkind:
+    #!/usr/bin/env bash
+    kind delete cluster --name {{CLUSTER_NAME}}
+
+    if docker inspect kind-registry &>/dev/null; then
+        echo "Stopping container registry"
+        docker kill kind-registry
+        docker rm kind-registry
+    fi
+
+
+###############################################################################
+# TL/DR actions.  These don't exist, other than for convenience to run the
+# entire flow without splitting across multiple "org" terminals.
+###############################################################################
+
+start-network:
+    just start org0
+    just start org1
+    just start org2
+
+# Shut down the test network and remove all certificates
+destroy:
+    #!/usr/bin/env bash
+    rm -rf organizations/org0/enrollments && echo "org0 enrollments deleted"
+    rm -rf organizations/org1/enrollments && echo "org1 enrollments deleted"
+    rm -rf organizations/org2/enrollments && echo "org2 enrollments deleted"
+    rm -rf organizations/org0/chaincode   && echo "org0 chaincode packages deleted"
+    rm -rf organizations/org1/chaincode   && echo "org1 chaincode packages deleted"
+    rm -rf organizations/org2/chaincode   && echo "org2 chaincode packages deleted"
+
+    rm -rf channel-config/organizations   && echo "consortium MSP deleted"
+    rm channel-config/{{CHANNEL_NAME}}_genesis_block.pb && echo {{CHANNEL_NAME}} " genesis block deleted"
+
+    kubectl delete ns org0 --ignore-not-found=true
+    kubectl delete ns org1 --ignore-not-found=true
+    kubectl delete ns org2 --ignore-not-found=true
+
+# Check that all network services are running
+check-network:
+    scripts/check-network.sh
+
+
+###############################################################################
+# Test Network
+###############################################################################
```

Comment:
I think it would be worth a small line here to check that the org exists... just check that the `organization/<org>` directory exists... first.